### DISCUSSION TITLE
Getting a 500 Unexpected Problem when getting zone forecast's

### DISCUSSION THREAD

**Discussion by carlosga01** [weather-gov/api#219] [Q&A]
I have been trying to hit this endpoint `/zones/{type}/{zoneId}/forecast` but keep getting a 500. I get the 500 whether I try to hit it from the swagger api examples (https://www.weather.gov/documentation/services-web-api#/) or from my application providing the required headers listed in the docs. Any ideas why this might be? I had seen this same issue weeks ago and it doesn't seem to have been resolved.

**Comment by scadergit** [comment]
Do you have a correlation I'd from the error?

**Comment by carlosga01** [comment]
Here's an example response body:
```
{
    "correlationId": "23a051dd",
    "title": "Unexpected Problem",
    "type": "https://api.weather.gov/problems/UnexpectedProblem",
    "status": 500,
    "detail": "An unexpected problem has occurred.",
    "instance": "https://api.weather.gov/requests/23a051dd"
}
```

**Comment by scadergit** [comment]
Thanks, that helped me see your url, which requested a zone forecast product for a marine zone. There are no ZFP products for marine, so the API should have returned a 404, not 500. We'll update the API to catch that.
